packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -251,13 +257,14 @@ export const library = [
       makeDefinition(
         [
           frDict(
-            ["xDist", frDist],
-            ["yDist", frDist],
+            ["xDist", frSampleSetDist],
+            ["yDist", frSampleSetDist],
```

Comment:
Good point about it being a regression.

I really doubt anyone was using this - and if they were, like this, it seems like it's likely it was an accident. 

I think it's worth us going forward with this, mainly because we have so few users now. I could change the changeset to highlight it at least.